src/containers/pages/PlanAssignment/helpers/JurisdictionAssignmentForm/helpers.ts

```diff
@@ -25,7 +26,11 @@ export const getPayload = (
 ): Assignment[] => {
   const now = moment(new Date());
   let startDate = now.format();
-  const endDate = moment(selectedPlan.effectivePeriod.end).format();
+  let endDate = moment(selectedPlan.effectivePeriod.end).format();
+
+  if (PLAN_TEAM_ASSIGNMENT_EXPIRY_YEARS > 0) {
+    endDate = now.add(PLAN_TEAM_ASSIGNMENT_EXPIRY_YEARS, 'years').format();
+  }
```

Comment:
what's should happen when the team assignment is greater than than the plans end date i.e. `endDate < plans.effectivePeriod`

Reply:
The plan wouldn't be valid but it will be possible to see the teams assigned to the plan.